File: 프로그래머스/0/181847. 0 떼기/0 떼기.py
def solution(n_str):
    answer = ''
    index0 = 0
    for i in range(len(n_str)):
        if(n_str[i] != '0'):
            index0 = i
            break
    for i in range(index0, len(n_str), 1):
        answer += n_str[i]
    return answer

File: 프로그래머스/0/181847. 0 떼기/0 떼기.js
function solution(n_str) {
    var answer = '';
    var index = 0;
    for(let i = 0; i < n_str.length; i++){
        if(n_str[i] != '0'){
            index = i;
            break;
        }
    }
    for(let i = index; i < n_str.length; i++){
        answer+= n_str[i];
    }
    return answer;
}